lib/pages/splash_page/splash_page.dart

```diff
@@ -28,14 +33,27 @@ class _SplashPageState extends State<SplashPage> {
             tadabburService: ref.read(tadabburService),
             remoteConfigService: ref.read(remoteConfigService),
             habitDailySummaryService: ref.read(habitDailySummaryService),
+            sharedPreferenceService: ref.read(sharedPreferenceServiceProvider),
           );
         },
       ),
       onStateNotifierReady: (notifier, _) async {
-        await Future.delayed(const Duration(milliseconds: 500), () async {
-          await notifier.initStateNotifier();
-          Navigator.of(context).pushReplacementNamed(RoutePaths.routeMain);
-        });
+        Connectivity cn = Connectivity();
```

Comment:
Is there a possibility of an error occurring in this block? Should we add a try-catch block for this?

Reply:
I dont think we need to add try-catch block ndras. Since every API/service all under these lines are already had their own try-catch block inside. Wdyt ?

For connection checking, I don't think there'll be any error possibility (?)

Reply:
Amann